src/components/OtherButtons/OtherButtons.tsx

```diff
@@ -0,0 +1,30 @@
+import { faUpload} from '@fortawesome/free-solid-svg-icons';
+import { faDownload} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+
+
+type Props = {
+    variant: string;
+  };
+  
+  const OtherButton = ({variant}:Props) => {
```

Comment:
Nazwa komponentu nie wskazuje na to co on faktycznie robi, a powinna sugerować to co jest wyświetlane. np. CircularButton -> Okrągły przycisk, a tutaj OtherButton nie wiele nam mówi